Docs/Configuration.tex

```diff
@@ -6954,6 +6940,119 @@ \subsubsection{Configuration}
   some other systems. \medskip
 \end{itemize}
 
+\subsection{OpenVariableRuntimeDxe}\label{emunvram}
+
+Provides in-memory emulated NVRAM implementation. This can be useful on systems with fragile
+(e.g. MacPro5,1, see discussion linked from this \href{https://forums.macrumors.com/threads/opencore-on-the-mac-pro.2207814/page-425?post=30945127#post-30945127}{forum post})
```

Comment:
You could tighten the link here as follows:
https://forums.macrumors.com/posts/30945127

Reply:
They're definitely not always tightened elsewhere. I didn't know that format, I'll make the change. Thanks.

Reply:
It must just be that I was at some point thinking all such external links _SHOULD_ be explictly written and not hyperlinked.

The same format can be used for the Applelife.ru link further below btw as they use the same software as MacRumors.
Both the long and short format do not currently work for AppleLife ... server is down or SSL cert is invalid/revoked.

Reply:
I just checked again, and it's almost exclusively done in the way I've done it, in the main `Configuration.pdf` file (which fits with the original intended usage of hyperlinks, btw). So if you believe it should be done another way, I think that's a separate battle for you to fight in a separate issue! ;-)

But thanks for the short link, equally for AppleLife, and yes I noticed that was down (at least for us to access? not sure).